share/smack/top.py

```diff
@@ -345,8 +357,19 @@ def llvm_frontend(args):
 def clang_frontend(args):
   """Generate Boogie code from C-language source(s)."""
 
-  bitcodes = []
   compile_command = default_clang_compile_command(args)
+  clang_frontend_helper(compile_command, args)
+
+def objc_clang_frontend(args):
+  """Generate Boogie code from Objective-C language source(s)."""
+
+  compile_command = objc_clang_compile_command(args)
+  clang_frontend_helper(compile_command, args)
+
+def clang_frontend_helper(compile_command, args):
```

Comment:
I think maybe it's better to name this function as something like `link-bc-files`.

Reply:
I agree. It looks like this function doesn't have anything to do with clang, and that it will be used by other languages. It looks like the llvm frontend and the rust frontend don't use it though, so I propose `default_link_bc_files`.